pstress/pstress-cluster-grp.cfg

```diff
@@ -0,0 +1,20 @@
+[node1]
+socket = /tmp/node1/node1_socket.sock
+logdir = /tmp
+threads = 10
+run = true
+user = root
+
+[node2]
+socket = /tmp/node2/node2_socket.sock
+threads = 10
```

Comment:
If we do not use regular threads number in multi-node group repl I would add this info in "Number of threads to use. Default: 10.  " option desctription

Reply:
The default value of THREADS is set to 10 in all the configuration files for eg
pstress-run.conf
pstress-run-PXC80.conf
pstress-run-PXC57.conf
pstress-run-rocksdb.conf

So, we are keeping the same behaviour in multi-node runs as well.

Reply:
I will add a description section in the cfg file which will cover this

Reply:
I mean that if we enable multi-node GR test for PS we will not use value from "pstress-run.conf"  THREADS. We will use it from "[pstress/pstress-cluster-grp.cfg]". And it would be good to have information in pstress-run.conf that it's threads value will not be used for GR.